src/components/MovieInfo/MovieInfo.js

```diff
@@ -1,28 +1,39 @@
 
 import React, { Component } from 'react';
-import testDetails from '../MovieInfo/mockMovieView';
+import { fetchSingleMovieDetails } from '../../apiCalls';
 import './MovieInfo.css';
 
 class MovieInfo extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movie: testDetails.movie
-    };
+  constructor() {
+    super();
+    this.state = {};
   }
 
-  render() {
-    const { movie } = this.state;
+  componentDidMount() {
+    fetchSingleMovieDetails(this.props.selectedMovieId)
+        .then(data => this.setState(data.movie))
+        .catch()
```

Comment:
Similar to my other comment.  Think about what you want to have happen in your application if anything fails fetching the data.  What will the user expect/need to see?